erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md

```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# E Invoicing under GST
+
+Under GST, all small and large businesses are supposed to register all sales invoices onto GST portal that falls under one of the following categories:
```

Comment:
all small and large businesses? isnt' it only for companies with turnover above 500 crore?

Reply:
Its mandatory for 100Cr+ businesses from 1st Jan. Also its going to be the same for all businesses from 1st April 2021.

Right now I have changed this to 100Cr+